### PR TITLE
CI: add charm4py test

### PR DESCRIPTION
**Pull request by matthiasdiener**

### CODE REVIEW

File: .github/workflows/charm4py.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: build-charm++
+      run: ./build charm4py netlrts-linux-x86_64 tcp -g -j4 --with-production
+    - name: build-charm4py
+      run: |
+        cd ..
+        git clone git clone https://github.com/UIUC-PPL/charm4py
```
Comment (evan-charmworks) [inline .github/workflows/charm4py.yml:16]: Duplicate `git clone`?

File: .github/workflows/charm4py.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        mkdir -p charm4py/charm_src/charm/bin && cp bin/charmrun charm4py/charm_src/charm/bin
+        mkdir -p charm4py/charm_src/charm/lib && cp lib/libcharm.so charm4py/charm_src/charm/lib
+        mkdir -p charm4py/charm_src/charm/include && cp -r -L include/* charm4py/charm_src/charm/include
+        
+        cd charm4py
+        export CHARM4PY_BUILD_CFFI=1
+        python3 setup.py build_ext --inplace
+        python2 setup.py build_ext --inplace
+        
+        python3 auto_test.py
```
Comment (evan-charmworks) [inline .github/workflows/charm4py.yml:28]: This line duplicates the test step below?

File: .github/workflows/charm4py.yml (hunk 1 above)
Comment (matthiasdiener) [reply .github/workflows/charm4py.yml]: Should be fixed now.

File: .github/workflows/charm4py.yml (hunk 2 above)
Comment (matthiasdiener) [reply .github/workflows/charm4py.yml]: Should be fixed now.

**Comment by rbuch** [comment]
This branch built successfully for me using `./build charm4py netlrts-linux-x86_64` on my machine.

**Comment by matthiasdiener** [comment]
Ready for review.

**Pull request merged**